Tests/OrdersTests/OrdersTests.swift

```diff
@@ -1,15 +1,192 @@
 import XCTVapor
+import Fluent
+import FluentSQLiteDriver
 @testable import Orders
+@testable import PassKit
```

Comment:
I need to create and encode some `internal` DTOs to test Passes and Orders API endpoints

Reply:
Well if they're `package` they should be accessible in the tests too right?

Reply:
Ok, I added the `PassKit` target to the dependencies of the test target (previously it was only a dependency of the main `Passes` and `Orders` targets) and now it doesn't require `@testable`.

For `Passes` and `Orders` I wouldn't remove the `@testable` attribute because there are some `internal`s that can be tested